src/FixedDataTable.js

```diff
@@ -712,8 +712,9 @@ var FixedDataTable = React.createClass({
     /*number*/ left,
     /*object*/ event
   ) {
-    var isFixed = !!this.state.headFixedColumns.find(function(column) {
-      return column.props.columnKey === columnKey;
+    // No native support in IE11 for find, findIndex, or includes, so using every.
+    var isFixed = !this.state.headFixedColumns.every(function(column) {
```

Comment:
Ah, didn't notice that some was native and supported! Thank you, that's much better - changed.